### PR TITLE
Correct cookie message language

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

Updates the cookie message from 

> Welcome to the Guardian. This site uses cookies, read our policy [here]()

to

> Welcome to the Guardian. This site uses cookies. Read our policy [here]().
## What is the value of this and can you measure success?

request from Mary

### CODE REVIEW

**Comment by JonNorman** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/ui/cookiesBanner.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ define([
                 euMessageCookie = cookies.get(EU_COOKIE_MSG);
             if (!euMessageCookie || euMessageCookie != 'seen') {
                 var link = 'https://www.theguardian.com/info/cookies',
-                    txt = 'Welcome to the Guardian. This site uses cookies, read our policy <a href="' + link + '" class="cookie-message__link">here</a>',
```
Comment (TBonnin) [inline static/src/javascripts/projects/common/modules/ui/cookiesBanner.js]: Are we not supposed to also remove the `here` and make `policy` the link? :)

Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/ui/cookiesBanner.js]: Generally having a link that is understandable standalone is the most ideal so 'Read our policy here' might make more sense as the link if changing that.

**Comment by sndrs** [comment]
Updated per multiple requests :)

> Welcome to the Guardian. This site uses cookies. Read [our policy]().

**Comment by gtrufitt** [comment]
👍

**Pull request merged**